//while guard failure

foo() {
  while(9) {}
}